### PR TITLE
assign data row split

### PR DESCRIPTION
**Pull request by msokoloff1**
allows users to assign data rows to splits in diagnostics

### CODE REVIEW

File: labelbox/schema/model_run.py [hunk 1]
```diff
@@ -180,6 +181,39 @@ def delete_model_run_data_rows(self, data_row_ids):
             data_row_ids_param: data_row_ids
         })
 
+    @experimental
+    def assign_data_rows_to_split(self,
+                                  data_row_ids,
+                                  split,
+                                  timeout_seconds=60):
+        valid_splits = ["TRAINING", "TEST", "VALIDATION"]
```
Comment (jtsodapop) [inline labelbox/schema/model_run.py:189]: **Thought:** Would it make sense to have this as an enum of Dataset or ModelRun? Unlikely, but if we add more to the splits or use this elsewhere

**Review by jtsodapop — APPROVED** [review]
LGTM, left one comment but not really necessary

File: labelbox/schema/model_run.py (hunk 1 above)
Comment (msokoloff1) [reply labelbox/schema/model_run.py:189]: It prob should be an enum. My thought was that 1) we aren't consistent with our usage of enums so this isn't inconsistent with our existing lack of a pattern and 2) if we require an enum it is another thing for users to import. I do want to chat about this with you and Kevin today if you have some time. Just more generally what we want to do.

Comment (jtsodapop) [reply labelbox/schema/model_run.py:189]: sgtm!

File: labelbox/schema/model_run.py [hunk 2]
```diff
@@ -180,22 +195,62 @@ def delete_model_run_data_rows(self, data_row_ids):
...
+        if split_value == DataSplit.UNASSIGNED.value:
+            raise ValueError(
+                f"Cannot assign split value of `{DataSplit.UNASSIGNED.value}`.")
+
+        valid_splits = filter(lambda name: name != DataSplit.UNASSIGNED.value,
+                              DataSplit._member_names_)
+
+        if split_value not in valid_splits:
+            raise ValueError(
+                f"split must be one of : `{valid_splits}`. Found : `{split}`")
```
Comment (kkim-labelbox) [inline labelbox/schema/model_run.py:215]: Capitalize `Split`?

**Review by kkim-labelbox — COMMENTED** [review]
A couple small nits. Otherwise, LGTM!

File: labelbox/schema/model_run.py (hunk 2 above)
Comment (msokoloff1) [reply labelbox/schema/model_run.py:215]: It is a variable name. I'll put tickets around it maybe to highlight that.

File: labelbox/schema/model_run.py [hunk 3]
```diff
@@ -180,22 +195,62 @@ def delete_model_run_data_rows(self, data_row_ids):
             data_row_ids_param: data_row_ids
         })
 
+    @experimental
+    def assign_data_rows_to_split(self,
+                                  data_row_ids: List[str],
+                                  split: Union[DataSplit, str],
+                                  timeout_seconds=60):
```
[the remark this answers is not in the document]
Comment (msokoloff1) [reply labelbox/schema/model_run.py:202]: will update

**Review by jtsodapop — APPROVED** [review]

**Review by kkim-labelbox — APPROVED** [review]

**Pull request merged**